rifle.py

```diff
@@ -88,6 +123,56 @@ def draw_target_tags(self, screen):
                         (enemy.position_[0] * self.DRAWING_SCALE_ - (width/2),
                         (enemy.position_[1] * self.DRAWING_SCALE_ + enemy.size_/2) + (self.target_tag_y_spacing_ * self.DRAWING_SCALE_)))
 
+    def draw_friendly_projectiles(self, screen):
+        for projectile in self.universe_.friendly_projectiles():
+            projectile_rect = pygame.Rect((projectile.position_[0]-projectile.size_/2) * self.DRAWING_SCALE_,
+                                     (projectile.position_[1]-projectile.size_/2) * self.DRAWING_SCALE_,
+                                     projectile.size_ * self.DRAWING_SCALE_,
+                                     projectile.size_ * self.DRAWING_SCALE_)
+            pygame.draw.rect(screen, self.projectile_color_, projectile_rect)
+
+        main_character = self.universe_.main_character_
+        main_character_rect = pygame.Rect((main_character.position_[0]-main_character.size_/2) * self.DRAWING_SCALE_,
+                                          (main_character.position_[1]-main_character.size_/2) * self.DRAWING_SCALE_,
+                                          main_character.size_ * self.DRAWING_SCALE_,
+                                          main_character.size_ * self.DRAWING_SCALE_)
+        pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
+
+class RifleProjectile(GameObject): 
+
+    def __init__(self, initial_position, target_position):
+        self.ID_ = self.create_ID()
+        self.position_ = initial_position
+        self.speed_ = 15
+        self.size_ = 5
+        self.velocity_ = self.calculate_trajectory(initial_position, target_position)
+
+    def calculate_trajectory(self, initial_position, target_position):
+        y2 = target_position[1]
+        y1 = initial_position[1]
+        x2 = target_position[0]
+        x1 = initial_position[0]
+
+        tx = x2 - x1
+        ty = y2 - y1
+        distance = math.sqrt(tx*tx+ty*ty)
+        rad = math.atan2(ty,tx)
```

Comment:
Trigonometry is not needed to calculate the x and y velocity. Doing it this way also requires you to have the funky
`- abs(self.velocity_[1])` code on line 167. 

Copy pasted from `basic_grunt.py`:
```
        main_character_position = self.universe_.main_character().position_
        x_distance = main_character_position[0] - self.position_[0]
        y_distance = main_character_position[1] - self.position_[1]
        distance = (x_distance**2 + y_distance**2)**(1/2)
        ...
        x_velocity = (x_distance * self.speed_) / distance
        y_velocity = (y_distance * self.speed_) / distance
        self.position_ = (self.position_[0]+x_velocity, self.position_[1]+y_velocity)
```

Adapt it for this section of code.